docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -261,6 +262,47 @@ public class GreetingServiceTest {
 ----
 <1> The `GreetingService` bean will be injected into the test
 
+== Applying Interceptors to Tests
+
+As mentioned above Quarkus tests are actually full CDI beans, and as such you can apply CDI interceptors as you would
+normally. As an example, if you want a test method to run within the context of a transaction you can simply apply the
+`@Transactional` annotation to the method and the transaction interceptor will handle it.
```

Comment:
So, since the tx interceptor supports plugging into async return types, we're a step closer to supporting async test methods. Now we _just_ have to write the right JUnit extension to delay test results for test methods that return async types such as `CompletionStage` :)